config/betterquesting/DefaultQuests.json

```diff
@@ -13337,42 +13352,897 @@
+          "taskID:8": "bq_standard:checkbox"
+        }
+      }
+    },
+    "199:10": {
+      "preRequisites:11": [
+        184
+      ],
+      "properties:10": {
+        "betterquesting:10": {
+          "autoclaim:1": 0,
+          "desc:8": "§6Sodium Hydroxide solution§r can be combined with §6Hydrogen Fluoride§r to get §6Sodium Fluoride Solution§r.\n\nThis has to be§b §cdistilled §rto get Sodium Fluoride.",
+          "globalshare:1": 0,
+          "icon:10": {
+            "Count:3": 1,
+            "Damage:2": 32046,
+            "ForgeCaps:10": {
+              "regeneration:arch:10": {
+                "arch_status:8": "NORMAL_ITEM",
+                "regenAmount:3": 0,
+                "skin:8": "NONE",
+                "skinType:8": "STEVE",
+                "trait:8": "regeneration:boring"
+              }
+            },
+            "OreDict:8": "",
+            "id:8": "gregtech:meta_dust"
+          },
+          "ignoresview:1": 0,
+          "ismain:1": 0,
+          "issilent:1": 0,
+          "lockedprogress:1": 0,
+          "name:8": "Sodium Fluoride",
+          "questlogic:8": "AND",
+          "repeat_relative:1": 1,
+          "repeattime:3": -1,
+          "simultaneous:1": 0,
+          "snd_complete:8": "minecraft:entity.player.levelup",
+          "snd_update:8": "minecraft:entity.player.levelup",
+          "tasklogic:8": "AND",
+          "visibility:8": "ALWAYS"
+        }
+      },
+      "questID:3": 200,
+      "rewards:9": {},
+      "tasks:9": {
+        "0:10": {
+          "autoConsume:1": 0,
+          "consume:1": 0,
+          "groupDetect:1": 0,
+          "ignoreNBT:1": 0,
+          "index:3": 0,
+          "partialMatch:1": 1,
+          "requiredItems:9": {
+            "0:10": {
+              "Count:3": 1,
+              "Damage:2": 32046,
+              "ForgeCaps:10": {
+                "regeneration:arch:10": {
+                  "arch_status:8": "NORMAL_ITEM",
+                  "regenAmount:3": 0,
+                  "skin:8": "NONE",
+                  "skinType:8": "STEVE",
+                  "trait:8": "regeneration:boring"
+                }
+              },
+              "OreDict:8": "",
+              "id:8": "gregtech:meta_dust"
+            }
+          },
+          "taskID:8": "bq_standard:retrieval"
         }
       }
     },
-    "188:10": {
+    "200:10": {
       "preRequisites:11": [
-        191
+        190
       ],
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "The §6Pistol §ris one of the guns you can make at LV tier. It uses pistol magazines as ammo, and has a reload time of 1.75 seconds.\n\nFor more info, look at the tooltip.",
+          "desc:8": "§6Alumina §rneeds to be roasted with §6Hydrofluoric Acid§r to make §bAluminium Trifluoride.§r\n\n§rNote: you will need Aluminium Trifluoride as well as Cryolite for the final process. So don\u0027t convert all of it into Cryolite right away.",
```

Comment:
Minor grammar thing: "so" is a coordinating conjunction like "and", and shouldn't usually be placed at the beginning of a sentence, so change the "...process. So..." starting the final sentence into "...process, so...".

Reply:
SHUT UP

Reply:
YOU ARE WORSE THAN BEANIE 
I DO NOT WANT TO HEAR THIS AFTER 8.5 HOURS OF STUDYING